relay-prosperoconv/Cargo.toml

```
@@ -0,0 +1,32 @@
+[package]
+name = "relay-prosperoconv"
+authors = ["Sentry <oss@sentry.io>"]
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "25.4.0"
+edition = "2021"
```

```suggestion
edition = "2024"
```

May need to update from master first.